docscan/session/create/check/constants.go

```diff
@@ -0,0 +1,6 @@
+package check
+
+const (
+	liveness = "LIVENESS"
```

Comment:
Should `liveness = "LIVENESS"` live in `docscan/constants/constants.go` with the other check types?
(or should the other check type constants move here?)

Reply:
Will reorganise the constants before release, after all the Doc Scan stuff has been added 👍